### PR TITLE
Update apache json

### PR DESCRIPTION
**Pull request by ericwholt**
Adds observiq log format. This formats log entries in JSON. This log format also adds fields that are not present in the default log format.
- Add parameter `log_format`
- Add observiq JSON log format parsing for access and error logs
- Additional fields added by observiq format
  - `http_x_forwarded_for`, `logid.request`, `logid.connection`

### CODE REVIEW

**Comment by schmikei** [comment]
Do you happen to have a log file that I could double check this from?

File: plugins/apache_http.yaml [hunk 1]
```diff
@@ -107,4 +130,57 @@ pipeline:
...
+
+  - id: error_json_parser
+    type: json_parser
+    timestamp:
+      parse_from: time
+      layout: '%Y-%m-%d %H:%M:%S.%s'
+    severity:
+      parse_from: log_level
+      mapping:
+        critical: crit
```
Comment (schmikei) [inline plugins/apache_http.yaml:167]: Looking at this, should we also include info? https://stackify.com/apache-error-log-explained/

File: plugins/apache_http.yaml [hunk 2]
```diff
@@ -36,14 +36,31 @@ parameters:
      - beginning
      - end
     default: end
+  - name: log_format
+    label: Log Format
+    description: |2
+      Default is unmodifed log_format settings for Apache HTTPD access.log and error.log
 
+      Observiq Log Format expects the following formats for the access and error logs. This format will generate a json log entry. This log_format can be modified to meet your requirements as long as key names do not change and it is valid json.
+      To setup edit apache2.conf. Add both log formats below the default log formats. Edit sites-available configurations: modify access log to use observiq format instead of combined.
```
Comment (jsirianni) [inline plugins/apache_http.yaml:45]: I think this should be re-worded to be more generic. Our instructions are for Ubuntu, the process is different on Centos.

```
To setup, edit Apache's configuration. Add both log formats to the logging section. On Debian, update all `sites-enabled` configurations to include `CustomLog ${APACHE_LOG_DIR}/access.log observiq`. On CentOS, update Apache's configuration to include `CustomLog "logs/access_log" observiq`.
```

File: plugins/apache_http.yaml (hunk 1 above)
Comment (ericwholt) [reply plugins/apache_http.yaml:167]: By default mapping includes info. We map the others as they are different than the default mappings.

**Review by jsirianni — APPROVED** [review]

**Review by schmikei — APPROVED** [review]

**Pull request merged**